docs/cluster_spec.md

```diff
@@ -257,6 +257,8 @@ spec:
     - 12.34.56.78/32
 ```
 
+Instead of listing all CIDRs, it is possible to specify a pre-existing [AWS Prefix List](https://docs.aws.amazon.com/vpc/latest/userguide/managed-prefix-lists.html) ID
```

Comment:
```suggestion
Instead of listing all CIDRs, it is possible to specify a pre-existing [AWS Prefix List](https://docs.aws.amazon.com/vpc/latest/userguide/managed-prefix-lists.html) ID.
```

Reply:
Updated that to include the `.`